pkgs/native_assets_cli/lib/src/code_assets/c_compiler_config.dart

```diff
@@ -10,13 +10,13 @@ import '../utils/map.dart';
 /// The configuration for a C toolchain.
 final class CCompilerConfig {
   /// Path to a C compiler.
-  late final Uri? compiler;
+  late final Uri compiler;
```

Comment:
@dcharkes you recently brought up the question why I named it `config.codeConfig` and the class `CodeConfig` and we said we could drop "config" iff we structured the `LinkConfig` differently.

=> The same applies here, would we want to rename `CCompilerConfig` -> `CCompiler`?

Reply:
I think the class name should still include config. But the fields not. I have a CL that does that, but I'm not fully happy with how it looks, maybe it feels more natural to do `config.codeConfig.androidConfig.ndkVersion` rather than `config.code.android.ndkVersion`, I'll play around with it and send it out as draft PR for us to decide what we want.